### ISSUE TITLE
refactor scripts to use error function

### ISSUE THREAD

**Issue by j-ogas** [hpc/charliecloud#451]
We have a lot of repetitive `exit` and `stdout` redirection in our scripts that would be handled more elegantly with a function:

**Comment by j-ogas** [comment]
Was this addressed or partially addressed with #698?

**Comment by reidpr** [comment]
Yes, I think this issue is now “refactor the other scripts to use it”.
